package.json

```diff
@@ -21,5 +21,12 @@
   },
   "devDependencies": {
     "coffeelint": "^1.9.7"
+  },
+  "consumedServices": {
+    "file-icons": {
```

Comment:
I think we should maybe call the service `tree-view.file-icons`, to promote the convention of using package names as service namespaces.